### PR TITLE
Create allReleasesChangeLog.md

### PR DESCRIPTION
**Pull request by lpista**
[RDAPI-21101] - Create a functionality change log for all releases since 7.5.3

### CODE REVIEW

File: content/en/allReleasesChangeLog.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+### [Version 7.5.3 SP9](#7.5.3SP9)
+### [Version 7.5.3 SP8](#7.5.3SP8)
+### [Version 7.5.3 SP7](#7.5.3SP7)
+### [Version 7.5.3 SP6](#7.5.3SP6)
+### [Version 7.5.3 SP5](#7.5.3SP5)
+### [Version 7.5.3 SP4](#7.5.3SP4)
+### [Version 7.5.3 SP3](#7.5.3SP3)
+### [Version 7.5.3](#7.5.3)
+### [Version 7.5.3 SP2](#7.5.3SP2)
+### [Version 7.5.3 SP1](#7.5.3SP1)
```
Comment (dshielaxway) [inline content/en/allReleasesChangeLog.md:27]: Should these appear before 7.5.3?

Comment (lpista) [reply content/en/allReleasesChangeLog.md:27]: I know it looks weird but 7.5.3 looks like it was released after SP1 and 2... 
This is from the 7.5.3 release notes:
"The fixes for issues included in API Gateway v7.5.2 SP 1 and SP 2 are also included in API Gateway 7.5.3"

File: content/en/allReleasesChangeLog.md [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+---------------
```
Comment (dshielaxway) [inline content/en/allReleasesChangeLog.md:28]: Suggestion: I'm wondering if changes should be grouped by product; API Gateway, API Manager, Analytics, PolicyStudio, ConfigurationStudio.
Thoughts?

File: content/en/allReleasesChangeLog.md (hunk 1 above)
Comment (dshielaxway) [reply content/en/allReleasesChangeLog.md:27]: Very odd. Perhaps 7.5.3 was re-released. Maybe Natalia knows.

File: content/en/allReleasesChangeLog.md (hunk 2 above)
Comment (dshielaxway) [reply content/en/allReleasesChangeLog.md:28]: Overall, great to have all this detail in one place. Thanks Luis.

I do think that some of the entries are a little vague though, and probably require slight elaboration to give some context. For example, "API Manager Remote host updates" could be "API Manager Remote Hosts aligned with PolicyStudio functionality."

Not sure if it's worth expanding on "29 issues fixed" I guess they have the link to the release notes, so if they are interested they can click through.

May be worth getting feedback from PSO too.

File: content/en/allReleasesChangeLog.md [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+## Version [7.7.20200930](https://axway-open-docs.netlify.app/docs/apim_relnotes/20200930_apimgr_relnotes/) {#7.7.20200930}
+### Highlights
+* API Manager Remote host updates
+* YAML-based configuration store (Technical preview)
+* Database integration updates
+* User membership in multiple organizations
+* New API Management REST API version 1.4
+* 29 issues fixed
```
Comment (JohnD-Axway) [inline content/en/allReleasesChangeLog.md:36]: Does webliv have the list of issues fixed?
Can we link to them anywhere?

Comment (JohnD-Axway) [reply content/en/allReleasesChangeLog.md:36]: Nevermind, I see Dave already said above!

Comment (lpista) [reply content/en/allReleasesChangeLog.md:36]: yes the issues fixed are in the release notes

File: content/en/allReleasesChangeLog.md (hunk 2 above)
Comment (lpista) [reply content/en/allReleasesChangeLog.md:28]: at the beginning i was putting all the issues fixed and then realized that the document was gonna be huge and also, they are all in the release notes.

**Pull request merged**